UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowWeeklyFilesList.cshtml

```diff
@@ -22,10 +22,7 @@
                     <td id="filename_@i">@Model[i].FileDescription</td>
                     <td id="filesize_@i">@Model[i].FileSizeinKB (@Model[i].FileExtension)</td>
                     <td class="td-3 justify-content-center" id="download_@i">
-                        @using (Html.BeginForm("DownloadWeeklyFile", "NoticesToMariners", FormMethod.Post, new {id = "frmDownloadFile", target = "_blank" }))
-                        {
-                            <a target="_blank" href="@Url.Action("DownloadWeeklyFile", "NoticesToMariners", new { fileName = @Model[i].Filename, batchId = @Model[i].BatchId, mimeType = @Model[i].MimeType })">Download</a>;
-                        }
+                        <a target="_blank" href="@Url.Action("DownloadWeeklyFile", "NoticesToMariners", new { fileName = @Model[i].Filename, batchId = @Model[i].BatchId, mimeType = @Model[i].MimeType })">Download</a>
```

Comment:
It seems @using block is removed , is it overwritten during main merge or any specific reason to change it.?

Reply:
@using was not in use, suggested by avinash to remove it